fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -8,7 +8,12 @@ function addChild()
 
     var mixin = $("#new_mixin").val();
 
-    var newURI = $('#main').attr('resource') + (!id ? "" : "/" + id);
+    var base = $('#main').attr('resource');
+
+    // Remove trailing '/' if it exists
+    var cleanedBase = base.lastIndexOf('/') == base.length - 1 ? base.substr(0, base.length - 1) : base;
```

Comment:
I do not believe the `resource` can currently find itself in a double '/' situation... but I can add a `while-ends-with-slash` function to prevent such an issue in the future.